docs/_includes/menu-button.html

```diff
@@ -509,17 +510,17 @@ <h3 id="menu-button-fixed-width">Fixed Width Menu Button</h3>
     </div>
 </span>
     {% endhighlight %}
-
+    <!--
```

Comment:
Hmm...how come this large block has so many updates but is commented out?

Reply:
Ah, I was just shuffling a few things around and I forgot to delete that commented section. You can ignore it.

Reply:
Oh, I remember now, it's because I added this line at the top of menu button: 

> NOTE: The actual button can be switched out for a variety of other expandable button styles.

However, I may reinstate the form themed and borderless examples within menu button if it feels clearer.